### ISSUE TITLE
Focus doesn't return to buffer

### ISSUE THREAD

**Issue by paulhenrich** [alloy/macvim#32]
I'm using a build of the most recent split-browser on Snow Leopard.  On my build, keyboard focus doesn't return to the buffer after a new file is opened from the split-browser using the mouse or when the application receives focus from a cmd-tab.

Here's vim --version:
```VIM - Vi IMproved 7.3 (2010 Aug 15, compiled Jul 26 2011 16:01:56)
MacOS X (unix) version
Included patches: 1-260
Compiled by paulhenrich@mars.local
Huge version with MacVim GUI.  Features included (+) or not (-):
+arabic +autocmd +balloon_eval +browse ++builtin_terms +byte_offset +cindent 
+clientserver +clipboard +cmdline_compl +cmdline_hist +cmdline_info +comments 
+conceal +cryptv +cscope +cursorbind +cursorshape +dialog_con_gui +diff 
+digraphs +dnd -ebcdic +emacs_tags +eval +ex_extra +extra_search +farsi 
+file_in_path +find_in_path +float +folding -footer +fork() +fullscreen 
-gettext -hangul_input +iconv +insert_expand +jumplist +keymap +langmap 
+libcall +linebreak +lispindent +listcmds +localmap -lua +menu +mksession 
+modify_fname +mouse +mouseshape +mouse_dec -mouse_gpm -mouse_jsbterm 
+mouse_netterm -mouse_sysmouse +mouse_xterm +multi_byte +multi_lang -mzscheme 
+netbeans_intg +odbeditor +path_extra +perl +persistent_undo +postscript 
+printer +profile +python -python3 +quickfix +reltime +rightleft +ruby 
+scrollbind +signs +smartindent -sniff +startuptime +statusline -sun_workshop 
+syntax +tag_binary +tag_old_static -tag_any_white -tcl +terminfo +termresponse
 +textobjects +title +toolbar +transparency +user_commands +vertsplit 
+virtualedit +visual +visualextra +viminfo +vreplace +wildignore +wildmenu 
+windows +writebackup -X11 -xfontset +xim -xsmp -xterm_clipboard -xterm_save 
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
      user exrc file: "$HOME/.exrc"
  system gvimrc file: "$VIM/gvimrc"
    user gvimrc file: "$HOME/.gvimrc"
    system menu file: "$VIMRUNTIME/menu.vim"
  fall-back for $VIM: "/Applications/MacVim.app/Contents/Resources/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_MACVIM -Wall -Wno-unknown-pragmas -pipe  -DMACOS_X_UNIX -no-cpp-precomp  -g -O2 -D_FORTIFY_SOURCE=1  
Linking: gcc   -L.  -L.       -L/usr/local/lib -o Vim -framework Cocoa -framework Carbon      -lncurses  -liconv -framework Cocoa    -fstack-protector -L/usr/local/lib  -L/Users/paulhenrich/perl5/perlbrew/perls/perl-5.14.0/lib/5.14.0/darwin-2level/CORE -lperl -lm -lutil -lc -framework Python   -lruby.1.9.1 -lobjc -L/Users/paulhenrich/.rvm/rubies/ruby-1.9.2-p180/lib 

```
```

**Comment by jeffmax** [comment]
I am having the same issue on Leopard.

**Comment by paulhenrich** [comment]
I accidentally wrote Leopard, but I'm actually on Snow Leopard, 10.6.8. I've updated the original post.

**Comment by jeffmax** [comment]
Woops, me too. I lose track of the version names. I am on 10.6.7.

**Comment by jeffkreeftmeijer** [comment]
For what it's worth, same issue on Lion here.

**Comment by paulhenrich** [comment]
This appears to be fixed as of 6558ecc883f37756401ebb109ca5d5448d0b1098 (possibly earlier)
